src/components/reusable/dropdown/dropdown.ts

```diff
@@ -16,6 +16,13 @@ import downIcon from '@carbon/icons/es/chevron--down/24';
 import errorIcon from '@carbon/icons/es/warning--filled/24';
```

Comment:
I think we need to use a smaller error icon here, try 16px size.

Reply:
Changed it to 16px